chain/events/filter/index.go

```diff
@@ -567,11 +775,6 @@ func (ei *EventIndex) CollectEvents(ctx context.Context, te *TipSetEvents, rever
 				addressLookups[ev.Emitter] = addr
 			}
 
-			tsKeyCid, err := te.msgTs.Key().Cid()
```

Comment:
good catch btw, for _each_ event, for _each_ element of an event, we're encoding the tipset key 🤦

Reply:
Done.